### PR TITLE
Use log.Info() in testInitializer.go

### PR DESCRIPTION
**Pull request by camilbancioiu**
Replace calls to `fmt.Println()` with calls to `log.Info()` or `log.Error()` in the widely-used `testInitializer.go`.
This change allows integration tests to selectively suppress blocks from being rendered to `stdout`, as well as other test-related information. Use the log-level string `integrationtests:NONE` to completely disable these logs.

### CODE REVIEW

File: integrationTests/testInitializer.go [hunk 1]
```diff
@@ -840,7 +840,7 @@ func AdbEmulateBalanceTxSafeExecution(acntSrc, acntDest state.UserAccountHandler
 	err := AdbEmulateBalanceTxExecution(accounts, acntSrc, acntDest, value)
 
 	if err != nil {
-		fmt.Printf("Error executing tx (value: %v), reverting...\n", value)
+		log.Error("Error executing tx (value: %v), reverting...\n", value)
```
Comment (SebastianMarian) [inline integrationTests/testInitializer.go:843]: \n could be removed

File: integrationTests/testInitializer.go [hunk 2]
```diff
@@ -1339,18 +1339,18 @@ func DisplayAndStartNodes(nodes []*TestProcessorNode) {
 		pkTxBuff, _ := n.OwnAccount.PkTxSign.ToByteArray()
 		pkNode := n.NodesCoordinator.GetOwnPublicKey()
 
-		fmt.Printf("Shard ID: %v, pkNode: %s\n",
+		log.Info(fmt.Sprintf("Shard ID: %v, pkNode: %s\n",
```
Comment (SebastianMarian) [inline integrationTests/testInitializer.go:1342]: \n could be removed

File: integrationTests/testInitializer.go [hunk 3]
```diff
@@ -1339,18 +1339,18 @@ func DisplayAndStartNodes(nodes []*TestProcessorNode) {
...
 			n.ShardCoordinator.SelfId(),
-			TestValidatorPubkeyConverter.Encode(pkNode))
+			TestValidatorPubkeyConverter.Encode(pkNode)))
 
-		fmt.Printf("skTx: %s, pkTx: %s\n",
+		log.Info(fmt.Sprintf("skTx: %s, pkTx: %s\n",
```
Comment (SebastianMarian) [inline integrationTests/testInitializer.go:1346]: \n could be removed

File: integrationTests/testInitializer.go [hunk 4]
```diff
@@ -145,7 +145,7 @@ func CreateMessengerWithKadDht(initialAddr string) p2p.Messenger {
 
 	libP2PMes, err := libp2p.NewNetworkMessenger(arg)
 	if err != nil {
```
[the remark this answers is not in the document]
Comment (camilbancioiu) [reply integrationTests/testInitializer.go:147]: Done.

File: integrationTests/testInitializer.go (hunk 1 above)
Comment (camilbancioiu) [reply integrationTests/testInitializer.go:843]: Removed.

File: integrationTests/testInitializer.go (hunk 2 above)
Comment (camilbancioiu) [reply integrationTests/testInitializer.go:1342]: Removed.

File: integrationTests/testInitializer.go (hunk 3 above)
Comment (camilbancioiu) [reply integrationTests/testInitializer.go:1346]: Removed.

**Review by SebastianMarian — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Pull request merged**